### PR TITLE
Update fw-input.css for Hacktober

### PR DESCRIPTION
**Pull request by aachh**
The input now has different border colors depending on its state.

### CODE REVIEW

File: src/components/fw-input/fw-input.css [hunk 1]
```diff
@@ -63,3 +63,19 @@
   border-radius: 2px;
   background-color: #FFFFFF;
 }
+.input .input-box :hover {
```
Comment (hemchander23) [inline src/components/fw-input/fw-input.css]: Thanks. Just a minor correction here. The space between `.input-box` and `:hover` needs to be removed :)

File: src/components/fw-input/fw-input.css [hunk 2]
```diff
@@ -63,3 +63,19 @@
...
+  box-sizing: border-box;
+  height: 30px;
+  width: 400px;
+  border: 1px solid #183247;
+  border-radius: 2px;
+  background-color: #FFFFFF;
+}
+.input .input-box :focus {
```
Comment (hemchander23) [inline src/components/fw-input/fw-input.css]: Same as the above comment. Can we remove the space 😀

**Comment by aachh** [comment]
Sure, that was a typo 😬
Also thanks for the opportunity to contribute, this is one of my first ever pull requests, I hope I did it right 😂

**Comment by aachh** [comment]
fixed 😅

**Comment by hemchander23** [comment]
You are welcome and it looks good now.

Don't forget to drop your name below so that I can add it to the contributor list - if not I will add just the ID 😀Don't forget to star the repo if you found it useful. 

Keep an eye on the repo for more contribution opportunities! ⚡️

**Comment by aachh** [comment]
Thanks, it was a pleasure to contribute! I'm @aachh, at your service!

**Pull request merged**